app/views/admin/enterprises/form/_business_details.html.haml

```diff
@@ -37,14 +37,16 @@
     = f.label :terms_and_conditions, t('.terms_and_conditions')
     %i.text-big.icon-question-sign{ "data-controller": "help-modal-link", "data-action": "click->help-modal-link#open", "data-help-modal-link-target-value": "terms_and_conditions_info_modal" }
 
-  .omega.eight.columns
+  - message = t('js.admin.enterprises.form.images.immediate_terms_and_conditions_removal_warning')
+  .omega.eight.columns#terms_and_conditions{data: { controller: 'terms-and-conditions', "terms-and-conditions-message-value": message, 'reflex-root': '#terms_and_conditions'}}
```

Comment:
I know you guys aren't very familiar with SR, so I thought I would link the relavent section in the docs to make it easier to understand what's happening.
https://docs.stimulusreflex.com/rtfm/morph-modes#scoping-page-morphs